app/rental_routes.py

```diff
@@ -0,0 +1,79 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+
+rentals_bp = Blueprint("rentals_bp", __name__, url_prefix="/rentals")
+
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def check_out():
+    request_body = request.get_json()
+    if "customer_id" not in request_body:
+        response_body = {"details": "Request body must include customer_id."}
+        return jsonify(response_body), 400
+    elif "video_id" not in request_body:
+        response_body = {"details": "Request body must include video_id."}
+        return jsonify(response_body), 400
+
+    video_id = request_body["video_id"]
+    customer_id = request_body["customer_id"]
+    video = Video.query.get(video_id)
```

Comment:
Even here we should be doing id format checks, and whether the record was deleted. Leaving the data in the database has far-reaching effects!